packages/synapse-react-client/src/lib/containers/CreateProjectModal.tsx

```diff
@@ -40,55 +41,42 @@ export const CreateProjectModal: React.FunctionComponent<
       }
     }
   }
-  return (
+
+  const dialogContent = (
     <>
-      <Modal
-        className="CreateProjectModal bootstrap-4-backport"
-        show={isShowingModal}
-        animation={false}
-        onHide={hide}
-        backdrop="static"
-      >
-        <Modal.Header closeButton>
-          <Modal.Title>Create a new Project</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <Form.Group>
-            <Form.Label htmlFor={'projectInput'}>Project Name</Form.Label>
-            <Form.Control
-              id="projectInput"
-              data-testid="projectInput"
-              value={projectName}
-              onChange={event => {
-                setProjectName(event.target.value)
-              }}
-              onKeyDown={(event: any) => {
-                if (event.key === 'Enter') {
-                  if (projectName !== '') {
-                    onCreateProject()
-                  }
-                }
-              }}
-            />
-          </Form.Group>
-          {errorMessage && <Alert variant="danger">{errorMessage}</Alert>}
-        </Modal.Body>
-        <Modal.Footer>
-          <div className="ButtonContainer">
-            <Button variant="default" onClick={onClose}>
-              Cancel
-            </Button>
-            <Button
-              variant="primary"
-              onClick={() => {
+      <TextField
+        id="projectInput"
+        label="Project Name"
+        value={projectName}
+        fullWidth
+        onChange={event => {
+          setProjectName(event.target.value)
+        }}
+        inputProps={{
+          onKeyDown: (event: React.KeyboardEvent<HTMLInputElement>) => {
+            if (event.key === 'Enter') {
+              if (projectName !== '') {
                 onCreateProject()
-              }}
-            >
-              Save
-            </Button>
-          </div>
-        </Modal.Footer>
-      </Modal>
+              }
+            }
+          },
+        }}
+      />
+      {errorMessage && <Alert severity="error">{errorMessage}</Alert>}
+    </>
+  )
+
+  return (
+    <>
+      <ConfirmationDialog
+        open={isShowingModal}
+        title="Create a new Project"
+        content={dialogContent}
+        confirmButtonText="Save"
+        onConfirm={onCreateProject}
+        onCancel={hide}
```

Comment:
Previously, clicking the modal close button called `hide`, which cleared the alert and text entered into the input before calling `onClose` whereas clicking "Cancel" only called `onClose`. 

Now, ConfirmationDialog will call `hide` both when clicking "Cancel" or when clicking the modal close button.